packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1314,8 +1315,12 @@ export class ContainerRuntime
 		return this.dataModelChangeRunner.run(callback);
 	}
 
+	/**
+	 * Indicates whether the container is in a state where it is able to send
+	 * ops (connected to op stream and not in readonly mode).
+	 */
```

Comment:
Future Direction (not blocking this PR):

@jason-ha do you anticipate a need to have 2 properties on IFluidParentContext, `canSendOps` (the semantics here) _and_ `connected` (which indicates Presence State can flow, for example)?

If so we should consider queueing up the legacy-alpha breaking changes there, if we want to reuse the name `connected`.  Or maybe you just add a new one `canSyncPresenceState` or something?